Sources/Order.swift

```diff
@@ -3,12 +3,42 @@ public enum Method: String {
 }
 
 public class Order {
+    public typealias Rules = [(nodeKey: String, rule: EngagementRule)]
     public let path: String
     public let method: Method
 
-    public init(path: String, method: Method = .GET) {
+    /// The rules for the nodes that should be fetched.
+    public let rulesOfEngagement: Rules?
```

Comment:
The whore RulesOf Engagement was an idea from @KrisBoonefaes and I like it but maybe we added it too soon now and make it a task for later. I will remove it from the code and created issue #64